evaluation/ragacli_pyproject/setup.py

```diff
@@ -0,0 +1,28 @@
+# make a setup.py for ragacli package
+
+from setuptools import setup, find_packages
+
+with open("README.md", "r", encoding="utf-8") as f:
+    long_description = f.read()
+
+setup(
+    name="raga_once",
```

Comment:
good name !  maybe `ragas_once`

Reply:
keep the direcotyr name save as your project name

Reply:
since the `raga-once` has been uploaded, I'm not sure if pypi supports changing the name of it